registry/kusama.json

```
@@ -1,4 +1,12 @@
 [
+     {
+        "address": "GqPDeJBexSsvEq6gBzrnMCn7KNYbzn9bX9ePCjEDN3EikoV",
+        "name": "Alex PromoTeam",
+        "image":  "https://raw.githubusercontent.com/GlobalSG/opengov-delegate-registry/0c1a101956dbd5c5ad30e1997d6974cf2e966b62/images/IMG_5814%204.JPG",
```

link must lead to our repo:
```suggestion
        "image":  "https://raw.githubusercontent.com/novasamatech/opengov-delegate-registry/master/images/IMG_5814%204.JPG",
```